ros2bag/ros2bag/verb/record.py

```diff
@@ -35,7 +35,7 @@ class RecordVerb(VerbExtension):
 
     def add_arguments(self, parser, cli_name):  # noqa: D102
         writer_choices = get_registered_writers()
-        default_writer = 'sqlite3' if 'sqlite3' in writer_choices else writer_choices[0]
+        default_writer = 'sqlite3' if 'sqlite3' in writer_choices else list(writer_choices)[0]
```

Comment:
```suggestion
        default_writer = 'sqlite3' if 'sqlite3' in writer_choices else next(iter(writer_choices))
```

Reply:
Looks fine to me. Added here and to reindex.py